cassis/typesystem.py

```diff
@@ -45,13 +45,15 @@
     "uima.cas.DoubleArray",
     "uima.cas.Sofa",
     "uima.cas.AnnotationBase",
+    "org.apache.uima.examples.SourceDocumentInformation",
```

Comment:
This type is not predefined,  you would need to load your typesystem and create the type before loading the cas with it, something around the lines of:

    from cassis import *

    with open('TypeSystem.xml', 'rb') as f:
        ts = load_typesystem(f)

    t = ts.create_type(name='org.apache.uima.examples.SourceDocumentInformation', supertypeName='uima.tcas.Annotation')
    ts.add_feature(t, name='uri', rangeTypeName='uima.cas.String')
    ts.add_feature(t, name="offsetInSource", rangeTypeName="uima.cas.Integer")
    ts.add_feature(t, name="documentSize", rangeTypeName="uima.cas.Integer")
    ts.add_feature(t, name="lastSegment", rangeTypeName="uima.cas.Integer")

    with open('737-v1.txt.xmi', 'rb') as f:
        cas = load_cas_from_xmi(f, typesystem=ts)